### DISCUSSION TITLE
Control vs disease

### DISCUSSION THREAD

**Discussion by mary77** [aertslab/scenicplus#103] [Q&A]
I have a huge pool of Control vs Disease multiome data. I would like to know what is the best way to process the data.
Do you recommend to run scenicplus on entire data then check the network differences at the end or should I subset the obj for each celltype first and then run scenicplus?

I really appreciate any suggestion.

**Comment by mary77** [comment]
Any idea?

**Reply by SeppeDeWinter** [reply]
Sorry, I lost track of this question.

There are multiple ways at approaching this.

In general I would recommend running it on the entire dataset and test for differences afterwards. The reason for this is that the inference in SCENIC+ strongly depends on background (e.g. if a TF is expressed in all cells of the control but not the disease you wont be able to pick up that TF when you run SCENIC+ separately on case and control).

On the other hand you might want to consider running it separately on case and control when there is quite a lot of variety within case/control and you want to study what underlies this intra-sample variety.

However, if you are only interested in studying the difference between case and control I would certainly run SCENIC+ on the entire dataset.

I hope this helps!


Best,

Seppe

**Comment by mary77** [comment]
Thank you so much Seppe,

Then subsetting by celltypes is not a good idea?
My follow up question is that if I run it on entire dataset then how can I compare case vs control. I mean after generating for example two different GRN is there any recommended tool to compare graphs?
